src/styles/main.scss

```diff
@@ -20,6 +20,9 @@
   text-align: center;
   vertical-align: center;
   user-select: none;
+  transition:
+    background 350ms,
+    color 350ms;
```

Comment:
It's good practice to specify a default transition property if you're only transitioning certain properties. This will prevent other properties from accidentally animating if they change.